onnxruntime/core/providers/cpu/tensor/upsample.cc

```diff
@@ -434,7 +435,7 @@ float CubicInterpolation1D(const T* Xdata,
   float result = 0;
   for (int i = 0, j = -1; i < static_cast<int>(CubicModeGridLength); i++, j++) {
     auto orig_data = GetDataForCoordinate(Xdata, x + j, y, input_height, input_width);
-    result += static_cast<float>(coeff_array[i] * orig_data);
+    result += static_cast<float>(coeff_array[i]/coeff_sum * orig_data);
```

Comment:
Is the cast needed anymore given you're dividing by a float now?

Reply:
right will get rid of it